src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class Ball extends ColorSupplier {

```

Comment:
```suggestion
public class Ball {
```
class Ball should not extend ColorSupplier